lib/studies/reply/adaptive_nav.dart

```diff
@@ -816,15 +838,31 @@ class _ReplyFab extends StatelessWidget {
                 style: Theme.of(context)
                     .textTheme
                     .headline5
-                    .copyWith(fontSize: 16),
+                    .copyWith(fontSize: 16, color: ReplyColors.black900),
               ),
           ],
         ),
       );
     } else {
       return FloatingActionButton(
         heroTag: 'Bottom App Bar FAB',
-        child: const Icon(Icons.create),
+        tooltip: onMailView ? 'Reply' : 'Compose',
+        child: AnimatedSwitcher(
+          duration: const Duration(milliseconds: 350),
+          transitionBuilder: (child, animation) => ScaleTransition(
+            child: child,
+            scale: animation,
+          ),
+          child: onMailView
+              ? Icon(
```

Comment:
nit: these Icons should each fit on one line. not sure that the key is necessary